tests/unit/test_blob.py

```diff
@@ -1542,10 +1542,16 @@ def test_download_as_string_w_response_headers(self):
 
     def test_download_as_string_w_hash_response_header_none(self):
```

Comment:
We should be renaming this method to correspond with its new semantics, e.g.:

```suggestion
    def test_download_as_string_w_hash_response_header_missing(self):
```